frontend/tests/features/collection/revision.test.ts

```diff
@@ -183,16 +188,23 @@ async function startRevision(page: Page): Promise<string> {
           { page }
         );
       } catch {
-        await page
-          .getByTestId(COLLECTION_ROW_WRITE_REVISION_ID)
-          .first()
-          .getByTestId(COLLECTION_VIEW_REVISION)
-          .click();
```

Comment:
this unconditionally wait for a suitable collection to appear til the max test timeout of 5 mins, which is why the test takes 30 mins to timeout in GHA